db/schema.rb

```diff
@@ -10,9 +10,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_10_17_185107) do
+ActiveRecord::Schema.define(version: 2022_01_13_074644) do
 
-  create_table "activations", charset: "utf8mb3", force: :cascade do |t|
+  create_table "activations", charset: "utf8", force: :cascade do |t|
```

Comment:
I don't know how this is translated to MySQL, but if it's literal, this is a bad idea.

See:

- https://adamhooper.medium.com/in-mysql-never-use-utf8-use-utf8mb4-11761243e434
- https://stackoverflow.com/a/64209801/2892716

Reply:
Oh, good catch thanks! I wonder why rails wants to set it to utf8, I didn't change that manually. According to the articles we better switch to `utf8mb4` instead of the current `utf8mb3`.